entities/library.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative '../autoload'
+
+# Creates object which represents Library
+# Helps manage the basic needs of Library
+class Library
+  include Statistics
+  include Database
+
+  attr_accessor :authors, :readers, :books, :orders
+
+  def initialize(authors = [], books = [], readers = [], orders = [])
+    @authors = authors
+    @books = books
+    @readers = readers
+    @orders = orders
+  end
+
+  # Library app must have an ability to add each of the entity to itself
+  def add_entity(entity)
+    case entity
+    when Author  then @authors << entity
+    when Book    then @books << entity
+    when Order   then @orders << entity
+    when Reader  then @readers << entity
+    else
+      raise ArgumentError
+    end
+
+    data :save
```

Comment:
`add_entity` method should have 1 responsibility, please remove saving data from here